src/Popover.svelte

```diff
@@ -48,16 +48,40 @@
     }
   };
 
+  const open = () => isOpen = true;
+  const close = () => isOpen = false;
+  const toggle = () => isOpen = !isOpen;
+
   onMount(() => {
     targetEl = document.querySelector(`#${target}`);
-    if (hover) {
-      targetEl.addEventListener('mouseover', () => isOpen = true);
-      targetEl.addEventListener('mouseleave', () => isOpen = false);
-    } else {
-      targetEl.addEventListener('click', () => isOpen = !isOpen);
+    switch (trigger) {
+      case 'hover':
+        targetEl.addEventListener('mouseover', open);
+        targetEl.addEventListener('mouseleave', close);
+        break;
+      case 'focus':
+        targetEl.addEventListener('focus', open);
+        targetEl.addEventListener('blur', close);
+        break;
+      default:
```

Comment:
i think this does not include dismisable one. Suggest to change like this.

```javascript
    default:
          targetEl.addEventListener('click', toggle);
          if (dismisable) target.addEventListener('blur', close);
          break;
```

Reply:
Great idea @eddie0329, thank you!  Updated.